addons/google_gmail/models/res_config_settings.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = 'res.config.settings'
+
+    google_gmail_client_identifier = fields.Char("Gmail Client Id", config_parameter='google_gmail_client_id')
```

Comment:
ok, good idea :)

Reply:
edit: not possible because of this
![screenshot_spect](https://user-images.githubusercontent.com/54316226/150999071-27819dbd-aa95-4289-880c-72f1d90a638a.png)

Reply:
Yet another reason doing this kind of string-based composition is a bad idea ... to cleanup in master.